src/css/style.css

```diff
@@ -1,32 +1,132 @@
 
 body {
-  background-color: bisque;
+  background-color: rgb(167, 133, 93);
+}
+
+h2 {
+  color: white;
+  font-size: medium;
+  font-weight: bold;
+  margin-left: 2vw;
 }
 
 nav {
   height: 10vh;
-  background-color: blue;
+  background-color: rgb(61, 61, 161);
 }
 
 main {
   height: 80vh;
+  overflow-y: scroll;
+}
+
+.logo-img {
+  width: 100%;
+  height: 84px;
+  background: no-repeat center/75% url("../css/img/pngwing.png");
+  object-fit: contain;
 }
 
 footer {
   height: 10vh;
 }
 
+h5 {
+  margin: 0 !important;
+  padding: 0 !important;
+}
+
+p {
+  padding-right: 0 !important;
+}
+
 ul {
   list-style-type: none;
   margin: 0;
   padding: 0;
   overflow: hidden;
 }
 
-ul > li.list-items > a{
+ul > li.list-items > a {
   text-decoration: none;
 }
 
+#closeIcon {
+  color: white;
+}
+
+#closeIcon:hover {
+  color: black;
+}
+
 .item-nav {
-  font-size: 1.5em;
-}
\ No newline at end of file
+  font-size: 2.5vw;
+}
+
+.btn-primary {
+  border: 2px solid black;
+  box-shadow: 6px 6px black;
+  font-weight: bold;
+  background-color: white;
+  border-color: black;
+  color: black;
+}
+
+.btn-primary:hover {
+  box-shadow: 4px 4px black;
+  background-color: black;
+  border-color: black;
+  color: white;
+}
+
+.btn-primary:active {
+  box-shadow: 2px 2px black;
+  background-color: white;
+  border-color: white;
+  color: white;
+}
+
+.btn-primary:focus {
+  background-color: grey;
+  border-color: black;
+  color: white;
+}
+
+.card-style {
+  box-shadow: 1px 8px 16px rgba(88, 88, 88, 0.48);
+}
+
+.card-text {
+  height: 144px;
+}
+
+.iconLike {
+  color: rgb(61, 61, 161);
+}
+
+.modal-popup {
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  height: 100vh;
+  width: 100vw;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
```

Comment:
- [ ] Design and placement of elements don't follow the given wireframe. Please compare the screenshot and the wireframe.

| Screenshot | Wireframe |
| --------------- | -------------- |
| ![Screenshot from 2021-12-25 20-49-42](https://user-images.githubusercontent.com/1642014/147390753-55e30b44-a920-4d9b-960f-76388accdbf3.png) | ![Wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-javascript/main/group-capstone/images/Comments.png?token=AAMQ4HW7ACRQ6ZHSLRR2QILB2CJZ4) |

Reply:
Thank you for your feedback @sinansevgi. The amends are made it in commit 87ee525.